### PR TITLE
[CI] Moved to github actions #82

### LINKED ISSUE #82: [CI] Move to github actions
Move to github actions as we did in other repos.

### PR DESCRIPTION
**Pull request by codesankalp**
Added github actions for django_loci.

Closes #82

### CODE REVIEW

**Comment by codesankalp** [comment]
Success Build URL: https://github.com/codesankalp/django-loci/actions/runs/651014903

**Comment by codesankalp** [comment]
@nemesisdesign Can you please review it?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install system packages
+      run: |
+        sudo apt update
+        sudo apt-get -qq -y install sqlite3 libsqlite3-dev libsqlite3-mod-spatialite gdal-bin
```
Comment (devkapilbansal) [inline .github/workflows/ci.yml:37]: @codesankalp Please break this in multiple lines to increase readability

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    - name: Install python dependencies
+      run: |
+        pip install -r requirements-test.txt
```
Comment (devkapilbansal) [inline .github/workflows/ci.yml:41]: Write this in single line only
```suggestion
      run: pip install -r requirements-test.txt
```

**Review by devkapilbansal — CHANGES_REQUESTED** [review]

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

**Comment by devkapilbansal** [comment]
Rebased the PR to `ci` branch and opened PR #88 to test the build

**Pull request merged**